main.ipynb

```diff
@@ -6,12 +6,62 @@
    "metadata": {},
    "source": [
     "# Earthly Insights: Uncovering the Planet's Patterns\n",
+    "### Analyzing raster data for the Earth's climate\n",
+    "Kamal Narra <br/>\n",
+    "Jayson Hurst <br/>\n",
+    "Nathan Tablang <br/>"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Table of contents:\n",
+    "1. [Introduction](#introduction)\n",
+    "2. [Data & Plan](#data)\n",
+    "3. [Part 1: Cleaning raster datasets](#part-1)\n",
+    "4. [Part 2: Visualizing the data](#part-2)<br/>\n",
+    "<span style=\"margin-left: 2px\">I  [Precipitation](#precipitation)<br/></span>\n",
+    "<span style=\"margin-left: 2px\">II  [Average Temperature](#avg)<br/></span>\n",
+    "<span style=\"margin-left: 2px\">III  [Water Vapor](#wv)<br/></span>\n",
+    "<span style=\"margin-left: 2px\">IV  [Wind Speed](#ws)<br/></span>\n",
+    "<span style=\"margin-left: 2px\">V  [Visualizing each feature on a map](#visual)<br/></span>\n",
+    "<span style=\"margin-left: 2px\">VI  [All Together](#all)<br/></span>\n",
+    "\n",
+    "5. [Part 3: Regression Analysis](#regression)<br/>\n",
+    "<span style=\"margin-left: 2px\">I  [Colinearity](#colinearity)<br/></span>\n",
+    "<span style=\"margin-left: 2px\">II  [Evaluation](#evaluation)<br/></span>\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction <a class=\"anchor\" id=\"introduction\"></a>\n",
     "\n",
-    "For our project, we analyzed raster data collected from a variety of sources to gain insights into the Earth's climate. Raster data is a type of spacial data that is represented as a grid of cells, with each cell containing a value that corresponds to a certain feature of the area, such as wind speed, precipitation, temperature, etc. Raster data is an ideal format for analyzing large areas, such as the whole planet, as it allows us to quickly identify patterns and trends across a wide range of locations.\n",
+    "Everything leads back to the roots: what we eat, what we touch, what we breathe, and anything in between could be traced back to human marvels of agricultural engineering.\n",
+    "From centuries of nomadic lifestyle to the modern predispostition of globalization and industrialization, caring more about farms, crops, soils, and others have never been more important than ever in the face of rapid climate change. Amongst many things, finding prime locations of crops could make great chain reactions as its productions could be used for the next goods to the other, up until the necessities that we interact with on a daily basis.\n",
     "\n",
```

Comment:
lines 44 and 45 i wrote myself -- just wanted to give a more defined motivation for the project